UnityProject/Assets/Scripts/Systems/CraftingV2/CraftingIngredient.cs

```diff
@@ -102,15 +94,50 @@ public bool WillInteract(HandApply interaction, NetworkSide side)
 			return false;
 		}
 
+		public bool WillInteract(InventoryApply interaction, NetworkSide side)
+		{
+			if (HasSimpleRelatedRecipe == false
```

Comment:
split these across multiple one lined if statements so they aren't written this awkwardly.